src/App.tsx

```diff
@@ -1,60 +1,139 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const getUserById = (userId: number): User | null => {
+  return usersFromServer.find(user => user.id === userId) || null;
+};
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [title, setTitle] = useState('');
+  const [isTitleWrong, setIsTitleWrong] = useState(false);
+  const [isUserWrong, setIsUserWrong] = useState(false);
+
+  const addTodo = (id: number) => {
+    const trimmedTitle = title.trim();
+
+    const foundedUser = usersFromServer.find(user => user.id === id);
```

Comment:
We have this function to find a user by ID already, no need to repeat, just use it